src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ClearNuGetLocalResourcesWindow.xaml.cs

```diff
@@ -30,5 +33,15 @@ private void CloseButton_Click(object sender, System.Windows.RoutedEventArgs e)
         {
             Close();
         }
+
+        private void ExecuteOpenExternalLink(object sender, ExecutedRoutedEventArgs e)
+        {
+            var hyperlink = (Hyperlink)e.OriginalSource;
+            if (hyperlink.NavigateUri != null)
+            {
+                Process.Start(hyperlink.NavigateUri.AbsoluteUri);
```

Comment:
Let's use `UIUtility.LaunchExternalLink` which has logic for opening the URL and ensuring failures are handled and don't crash VS: https://github.com/NuGet/NuGet.Client/blob/bb09daa40f1092776b4d9071ec0a2e176fa1e640/src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/UIUtility.cs#L13